src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -116,6 +143,24 @@ lastName|string|-|The last name of the employee
 middleInitial|string|-|The middle initial of the employee
 template|string|-|Hyperlink template to the resource
 
+### <a name="schema-exceptions"></a>Exceptions
+
+Name|Type|Format|Description
+---|---|---|---
+code|string|-|The system exception code defined for the exception. Example: BADCODE
+isBlocking|boolean|-|Defines whether the exception will prevent the Request from being submitted
+level|integer|-|The numeric level associated with the exception. Example: 99
+message|string|-|The user-facing message defined for the exception
+source|array|[Exception source](#schema-exceptionsource)|The source that has raised the exception
```

Comment:
Missing an attribute "parameters"? (Parameters for the messages of the exception)